src/core/packages/analytics/browser-internal/src/track_performance_measure_entries.ts

```diff
@@ -55,6 +58,13 @@ export function trackPerformanceMeasureEntries(analytics: AnalyticsClient, isDev
             );
           }
 
+          if (description?.length > MAX_DESCRIPTION_LENGTH) {
+            // eslint-disable-next-line no-console
+            console.error(
+              `The description for the measure: ${target} is too long. The maximum length is ${MAX_DESCRIPTION_LENGTH}. Strings longer than ${MAX_DESCRIPTION_LENGTH} will not be indexed or stored`
```

Comment:
Is there a discrepancy here when considering the added length of the prefix `[TTFMP]` resulting in the actual acceptable length of text entered being 249? Is that worth being denoted in the error message?

Reply:
Since the prefix is mandatory, the total allowed length remains 256 characters.

The console error will only be visible to developers, but I want to include a warning because any input exceeding 256 characters will be ignored by the indexer due to `ignore_above 256` , as long keywords tend to have issues in ES.

Reply:
console.warn probably fit better

Reply:
I'll update the mapping with the setting `ignore_above` set to 256
```
^{:type :keyword :ignore_above 256} [:doc] [:original-body :properties :meta :description]}

```
so the string above that number won't be indexed.

Reply:
The string will no be indexed above 256 but it's a good point. We could truncate and avoid sending the long text which won't be indexed

Reply:
updated here https://github.com/elastic/kibana/pull/212393/commits/1cb6e6656ec5d96c414fa90109872d10dc694474